src/Workspaces/Core/Portable/Shared/Collections/IntervalTree`1.cs

```diff
@@ -17,6 +17,7 @@ internal partial class IntervalTree<T> : IEnumerable<T>
         public static readonly IntervalTree<T> Empty = new IntervalTree<T>();
 
         protected Node root;
+        private readonly int count;
```

Comment:
I added this for debugging purposes.  It was really nice to be able to look at an interval tree and be able to tell how many elements were in it.